### PR TITLE
readMe updates; plus updated test for endpoints.JSON

### PR DESCRIPTION
**Pull request by captainread**
view the full readme: https://github.com/captainread/nc-games/blob/2f2ef1dd8af2ac262cfcb1ba227cb10a6b5b999e/README.md

Also includes my files relating to hosting. Didn't send a PR past you guys for the hosting bit (sorry, got in a muddle!), but you can check it now if you like? I'm live at: https://captainread-nc-games.cyclic.app/

### CODE REVIEW

**Review by hevdevs — APPROVED** [review]
Readme is lovely and clear! Just be careful in the instructions for the dotenv files, you don't want a semicolon on the end of the PGDATABASE=database_name - it's not actually SQL at this point, it's just a string so it would double up on the semi-colons and cause an error when trying to connect to the db.

I've also noticed an SQL error from looking at your hosted api which is causing the comment count for ALL reviews (/api/reviews) to equal 61, which is almost definitely the result of an oversight by me. 
This subquery is the cause, along with us missing a group by clause. I suspect it may be adding together all of the columns from all of the comments at the moment:

SELECT reviews.*, **(SELECT COUNT(*)::int FROM comments)** AS comment_count FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id

A slight refactor is needed, we can add a comment count column without a subquery like so - 

SELECT reviews.*, **COUNT(comments.comment_id)::INT AS comment_count** FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id
GROUP BY reviews.review_id

We don't need a subquery to select from comments because we are already doing a join to access the columns from the comment table

TLDR this is good to merge! Just need to update the SQL for getReviews - please put in an nchelp if you want to clarify anything I've said here, apologies for not spotting it sooner!
